relayer/pkg/chainlink/ocr2/client.go

```diff
@@ -159,11 +160,13 @@ func (c *Client) LatestTransmissionDetails(ctx context.Context, address string)
 }
 
 func (c *Client) ConfigFromEventAt(ctx context.Context, address string, blockNum uint64) (cc ContractConfig, err error) {
-	block, err := c.r.BlockByNumberGateway(ctx, blockNum)
+	block, err := c.r.BlockByNumberGateway(ctx, blockNum+1) // HAXX: temporary workaround for devnet 0.2.8
```

Comment:
`starknet-devnet` v0.2.9 should now be released 🎉 🤞

Reply:
Looks like this PR is rebased on an older version of my PR